src/murmur/Server.cpp

```diff
@@ -55,6 +55,16 @@
 
 #define UDP_PACKET_SIZE 1024
 
+#ifdef Q_OS_WIN
```

Comment:
What warning did you get for these?

setsockopt takes a const char * on Windows, and const void * in POSIX.

I don't see why we need the DWORD?

I don't see why we can't use char */const char * on both platforms.  Since it takes void */const void * on POSIX systems, that should suffice, no?

Existing code seems fine to me, but I don't know which warnings you are fixing here.